### PR TITLE
Added vanchor poolshare script

### PR DESCRIPTION
**Pull request by AhmedKorim**

### CODE REVIEW

**Review by drewstone — CHANGES_REQUESTED** [review]
LGTM, some styling left before merge

**Review by AtelyPham — COMMENTED** [review]
I got an issue about the connection between `docker` and `polkadot` API.

CleanShot 2022-08-19 at 23 42 29@2x

**Comment by AhmedKorim** [comment]
@AtelyPham  Seems this appeared after the percentage update

**Comment by drewstone** [comment]
Yes you need to provide a nonce value now to these functions.

**Review by AtelyPham — APPROVED** [review]
LGTM 👍 But the test failed. I wonder if it has any effect?

**Comment by drewstone** [comment]
Yep, need to make sure the CI is fixed before merge.

File: tests/integration_tests/substrate/wrapUnwrap.spec.ts [hunk 1]
```diff
@@ -51,11 +50,11 @@ async function createPoolShare(
   const id = nextAssetId.toNumber() - 1;
   const tokenWrapperNonce = await apiPromise.query.tokenWrapper.proposalNonce<Option<U32>>(name);
   const nonce  = tokenWrapperNonce.unwrapOr(new BN(0)).toNumber() + 1;
-  console.log(nonce);
   await polkadotTx(apiPromise, {
       section: 'sudo',
       method: 'sudo'
     },
+    // @ts-ignore
     [apiPromise.tx.tokenWrapper.setWrappingFee( 1,id ,nonce)], singer);
```
Comment (drewstone) [inline tests/integration_tests/substrate/wrapUnwrap.spec.ts:58]: Signer spelling `signer`

**Review by drewstone — CHANGES_REQUESTED** [review]
Need to fix spelling @AhmedKorim

**Review by drewstone — APPROVED** [review]
LGTM

**Pull request merged**